provider/datagen/src/lib.rs

```diff
@@ -113,9 +113,9 @@
 //! * `icu_compactdecimal`
 //! * `icu_displaynames`
 //! * `icu_relativetime`
-//! * `icu_singlenumberformatter`
+//! * `icu_singlenumberformatter` (excluded from 1.4 release)
```

Comment:
remove? we don't want this in the docs

Reply:
I didn't delete the line because the breadcrumbs are easier to follow if I say "excluded from 1.4 release" but if you feel strongly I will delete the line

Reply:
Well it's weird to have this in the docs. You can html comment them out in markdown, no?

Reply:
or just make them non-doc-comment lines by removing the `!`